crates/beerus-rpc/src/server.rs

```diff
@@ -108,6 +109,17 @@ trait BeerusApi {
         block_id: String,
         contract_address: String,
     ) -> Result<FieldElement>;
+
+    #[method(name = "starknet_declare_transaction")]
```

Comment:
Lets change this to match the spec here:
https://github.com/starkware-libs/starknet-specs/blob/8016dd08ed7cd220168db16f24c8a6827ab88317/api/starknet_write_api.json#L39

Reply:
Fixed!